tests/test_esss_fix_format.py

```diff
@@ -20,7 +20,7 @@ def input_file(tmpdir):
             bravo\s\t\s
             charlie
             \tdelta
-            echo\r
+            echo
```

Comment:
I just pushed a change where this is no longer required: now the file is opened with universal newline support, so Python takes care of the conversion for us.